aiven/client/cli.py

```diff
@@ -1197,25 +1197,30 @@ def service__migration_status(self):
             table_layout=layout,
         )
 
-    def _parse_redis_access_control(self):
+    def _parse_access_control(self):
         arg_vars = vars(self.args)
-        return {
+        result = {
             key: arg_vars[key].split()
             for key in {"redis_acl_keys", "redis_acl_commands", "redis_acl_categories"}
             if arg_vars[key] is not None
         }
+        for key in ["m3_group"]:
```

Comment:
Our style guide would say `in {"m3_group"}:`, though obviously here it doesn't really make a difference.

Reply:
It should probably only apply to `if key in` and not `for key in`, as the former leads to reasonable speed and the latter leads to unexpected ordering of the nested block in for.